### PR TITLE
Intmdb 548

### PR DESCRIPTION
**Pull request by Zuhairahmed**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: cdk/cdk-resources/alert-configuration/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+> AWS CDK [L1 construct] and data structures for the [AWS CloudFormation Registry] type `MongoDB::Atlas::AlertConfiguration` v1.0.0.
+
+[L1 construct]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html
+[AWS CloudFormation Registry]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html
+
+## Description
+
+Returns and edits the conditions that trigger alerts and how MongoDB Cloud notifies users. This collection remains under revision and may change. Refer to the legacy documentation for this collection in the following link.
+
+## API Docs
```
Comment (Zuhairahmed) [inline cdk/cdk-resources/alert-configuration/README.md:14]:
```suggestion
## MongoDB Atlas API Docs
```

File: cdk/cdk-resources/alert-configuration/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+For more information about the Api refer to: [Api Endpoints](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Alert-Configurations)
```
Comment (Zuhairahmed) [inline cdk/cdk-resources/alert-configuration/README.md:16]:
```suggestion
For more information about the API refer to: [Api Endpoints](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Alert-Configurations)
```

File: cdk/cdk-resources/alert-configuration/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# mongodb-atlas-alertconfiguration-cdktf
```
Comment (Zuhairahmed) [inline cdk/cdk-resources/alert-configuration/README.md:1]:
```suggestion
# @mongodb-awscdk/atlas-alertconfiguration
```
follow this format ^^ for all resources, MongoDB is the publisher with suffix of -awscdk/atlas-resourcename. see conversation in AWS CDK team slack channel

File: cdk/cdk-resources/alert-configuration/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+You can find more information about activating this type in the [AWS CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry-public.html).
+
+## Feedback
+
+This library is auto-generated and published to all supported programming languages by the [cdklabs/cdk-cloudformation] project based on the API schema published for `MongoDB::Atlas::AlertConfiguration`.
+
+* Issues related to this generated library should be [reported here](https://github.com/cdklabs/cdk-cloudformation/issues/new?title=Issue+with+%40cdk-cloudformation%2Fmongodb-atlas-alertconfiguration+v1.0.0).
+* Issues related to `MongoDB::Atlas::AlertConfiguration` should be reported to the [publisher](https://github.com/mongodb/mongodbatlas-cloudformation-resources/issues).
+
```
Comment (Zuhairahmed) [inline cdk/cdk-resources/alert-configuration/README.md:46]: add another bullet for feature requests link to here: https://feedback.mongodb.com/forums/924145-atlas?category_id=392596

File: cdk/cdk-resources/project/.projenrc.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  cdkVersion: '2.1.0',
+  defaultReleaseBranch: 'main',
+  majorVersion: 1,
+  releaseToNpm: true,
+  releaseTrigger: ReleaseTrigger.manual(),
+  docgen: true,
+  name: 'mongodb-atlas-project',
+  repositoryUrl: 'https://github.com/mongodb/mongodbatlas-cloudformation-resources.git',
+  sampleCode: false,
+  publishToPypi: {
```
Comment (SuperMohit) [inline cdk/cdk-resources/project/.projenrc.js:16]: @sangramPeerI L16-L28 needs to be removed. We are only publishing to npm.

Comment (sangramPeerI) [reply cdk/cdk-resources/project/.projenrc.js:16]: @SuperMohit  Ok. will remove it.

File: cdk/.projenrc.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+const { awscdk } = require('projen');
+const { ReleaseTrigger } = require('projen/lib/release');
+const project = new awscdk.AwsCdkConstructLibrary
+({
+    author: 'MongoDB',
```
Comment (Zuhairahmed) [inline cdk/.projenrc.js:5]: let's make the author: "MongoDBAtlas"

File: cdk/.projenrc.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    cdkVersion: '2.1.0',
+    defaultReleaseBranch: 'main',
+    majorVersion: 1,
+    releaseToNpm: true,
+    releaseTrigger: ReleaseTrigger.manual(),
+    docgen: true,
+    name: 'mongodb-atlas-project',
+    repositoryUrl: 'https://github.com/mongodb/mongodbatlas-cloudformation-resources.git',
+    sampleCode: false,
+    publishToPypi: {
```
Comment (Zuhairahmed) [inline cdk/.projenrc.js:17]: why are there references to other language package managers here?

File: cdk/cdk-resources/alert-configuration/README.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+[L1 construct]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html
+[AWS CloudFormation Registry]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html
+
+## Description
+
+Returns and edits the conditions that trigger alerts and how MongoDB Cloud notifies users. This collection remains under revision and may change. Refer to the legacy documentation for this collection in the following link.
+
+## MongoDB Atlas API Docs
+
+For more information about the API refer to: [Api Endpoints](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Alert-Configurations)
```
Comment (Zuhairahmed) [inline cdk/cdk-resources/alert-configuration/README.md:16]:
```suggestion
For more information about the API refer to: [API Endpoints](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Alert-Configurations)
```

**Pull request closed without merging**